AutomatedTesting/Gem/PythonTests/Prefab/tests/PrefabTestUtils.py

```diff
@@ -18,6 +19,110 @@
 import azlmbr.entity as entity
 import azlmbr.legacy.general as general
 
+def get_linear_nested_items_name(nested_items_name_prefix, current_layer):
```

Comment:
I found the use of the word "layer" confusing in this context, do you mean level?

Reply:
Agreed, think "level" in regards to hierarchies makes a bit more sense than layer.